source/javascripts/na/create_test_merchant_account.js

```diff
@@ -138,5 +163,27 @@ $(function() {
             $statusDiv.removeClass('hidden success notice');
             $statusDiv.addClass('error');
         }
-    } 
+    }
+    
+    window.onload = function() {
```

Comment:
Also with these lines, are these part of the removed control?

Reply:
Done!